openmetadata-ui/src/main/resources/ui/src/components/common/TestConnection/TestConnection.tsx

```diff
@@ -149,6 +150,15 @@ const TestConnection: FC<TestConnectionProps> = ({
     setTestStatus(undefined);
   };
 
+  const handleDeleteWorkflow = async (workflowId: string) => {
+    try {
+      const response = await deleteWorkflowById(workflowId, true);
+      setCurrentWorkflow(response);
```

Comment:
Don't we want to show a toast here? for success

Reply:
No, we just want to silently delete the workflow once it's finished.